### PR TITLE
Fixed case sensitive resource provider name

### PR DESCRIPTION
**Pull request by Ganeshrockz**
**Task name**: AzureMonitorV1

**Description**: There has been a recent change in the case of resource provider microsoft.insights. We were targeting to Microsoft.Insights till now and this broke the task.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** There is an open DTS in ADO

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/AzureMonitorV1/task.json [hunk 1]
```diff
@@ -195,7 +195,7 @@
...
+                "EndpointUrl": "$(endpoint.url)/subscriptions/{{subscriptionId}}/providers/Microsoft.AlertsManagement/alerts?api-version=2019-03-01{{#if ResourceGroupName}}&targetResourceGroup=$(ResourceGroupName){{/if}}{{#if monitorCondition}}&monitorCondition=$(monitorCondition){{#if timeRange}}&timeRange=$(timeRange){{/if}}{{#if alertState}}&alertState=$(alertState){{/if}}{{#if severity}}&severity=$(severity){{/if}}{{#equals '$(filterType)' 'resource' 1}}{{#if resource}}&targetResource=/subscriptions/{{subscriptionId}}/resourcegroups/{{ResourceGroupName}}/providers/{{resource}}{{/if}}{{/equals}}{{#equals '$(filterType)' 'alertrule' 1}}{{#if alertRule}}&alertRule=/subscriptions/{{subscriptionId}}/resourceGroups/{{ResourceGroupName}}/providers/microsoft.insights/{{alertRule}}{{/if}}{{/equals}}",
```
Comment (ds-ms) [inline Tasks/AzureMonitorV1/task.json:198]: Is this a regression introduced by the Azure Monitor team?

**Review by ds-ms — APPROVED** [review]
LGTM

**Comment by ds-ms** [comment]
I would suggest involving the Azure Monitor team and fix it at their end instead.

**Comment by Ganeshrockz** [comment]
> I would suggest involving the Azure Monitor team and fix it at their end instead.

Looks like they asked us to fix this as they don't have plans to revert this back

File: Tasks/AzureMonitorV1/task.json (hunk 1 above)
Comment (Ganeshrockz) [reply Tasks/AzureMonitorV1/task.json:198]: Nope. This looks like a change that is needed from their end.

**Pull request merged**